init.go

```diff
@@ -217,6 +221,8 @@ func DefaultServicesConfigTestnet() Services {
 		HubDomain:          "https://hub-staging.btfs.io",
 		EscrowDomain:       "https://escrow-staging.btfs.io",
 		GuardDomain:        "https://guard-staging.btfs.io",
+		ExchangeDomain:     "https://exchange-dev.bt.co",
+		SolidityDomain:     "https://grpc.trongrid.io:50052",
```

Comment:
can you add a migration inside `migrations.go` for people who did not have these fields (empty) to pick up? probably can check other fields to figure out testnet/mainnet

Reply:
fixed